barcode/codex.py

```diff
@@ -271,5 +271,72 @@ def get_fullcode(self):
         return super().get_fullcode()[1:]
 
 
+class MSI(Barcode):
+    """Initializes a new MSI (Modified Plessey) instance. The checksum is added automatically
+    when building the bars.
+
+    :parameters:
+        code : int or bytes or string
+            Code MSI int without checksum (added automatically).
+            Code MSI bytes without checksum. requires byteorder for int conversion
+            Code MSI string without checksum. requires encoding and byteorder for int conversion
+        writer : barcode.writer Instance
+            The writer to render the barcode (default: SVGWriter).
+        byteorder : string
+            'big' or 'little' ; to convert bytes to int
+        encoding : string
+            if set, convert bytes to string and use this as a label. defaults to utf-8
```

Comment:
Please use regular sphinx format here so docs render properly.

```suggestion
    :param code: Code MSI int without checksum (added automatically).
            Code MSI bytes without checksum. requires byteorder for int conversion
            Code MSI string without checksum. requires encoding and byteorder for int conversion
    :param writer: The writer to render the barcode (default: SVGWriter).
    :param byteorder: 'big' or 'little' ; to convert bytes to int
    :param encoding : if set, convert bytes to string and use this as a label. defaults to utf-8
```